### PR TITLE
new case study on connexin

### PR DESCRIPTION
**Pull request by sjnorth**
newly added, but will wait for the other 2 (rock choir and soaring falcon) to be ready before merging

### CODE REVIEW

File: app/pages/stories/stories/connexin.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          <h2 className='u-text-heading u-color-dark-gray u-text-l u-text-light u-margin-Tn'>
+            From humble beginnings to international reach
+          </h2>
+          <p className='para u-margin-Tn'>
+            Founded in the city of Hull as a bedroom hobby, Connexin provides hosted telephony services to both business and residential customers, while also owning and managing two UK fibre networks.
+            Connexin offers wireless broadband, without line rental, at speeds of up to 2 Gbps for businesses and up to 60 Mbps for residential customers. Before Connexin started there was only one internet provider in Hull.
+            The founders, Alex and Furqan, spotted a gap in the market and decided to offer an alternative option. They created the first metropolitan wireless and fibre hybrid network in the city.
+          </p>
+          <p className='para'>
+            ““We questioned why it cost so much to make telephone calls abroad. This curiosity led us to learning about how phone systems worked and how calls were carried,” says Alex.
```
Comment (jacobpgn) [inline app/pages/stories/stories/connexin.js]: ✂️ extra `"`

File: app/pages/stories/stories/connexin.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        title='Our monthly bill runs are quick and easy since switching to GoCardless'
+        author='Alex Yueng'
+        position='Co-founder'
+        company='Connexin'
+        benefits='Huge reduction in payment processing admin, increased flexibility in payment timings, better signup experience'
+        bgimgUrl='url(/images/stories/story-images/nutmeg/hero-nutmeg.jpg)'
+        sidebarLogoSrc='/images/stories/story-logos/connexin.png'
+        sidebarServices='Business telephony services, VoIP,  fibre & data services'
+        sidebarEmployees='23 employees'
+        sidebarPreviousProvider='CashBacs'
```
Comment (jacobpgn) [inline app/pages/stories/stories/connexin.js]: -> `Cashbacs`

File: app/pages/stories/stories/connexin.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+          <div className='story-page__image'>
+            <img src='/images/stories/story-images/connexin/connexin@2x.jpg' alt='' />
+          </div>
+          <p className='para'>
+            Alex ran the business during daytime while Furqan handled the night shift. As families and friends started using the alternative, they started to think that maybe they had a viable product.
+            Now Connexin is a ten year old business, still based in Hull, but with a view to expand to 20 more cities around the UK.
+            But how did they handle all their customer payments? Before switching to GoCardless, Connexin used CashBacs to process direct debits.
+          </p>
+          <p className='story-page__quote'>
+            “The process of invoicing was long and labour intensive; once a month we’d spend a whole day inputting numbers into the CashBacs software.
```
Comment (jacobpgn) [inline app/pages/stories/stories/connexin.js]: -> `Cashbacs`

File: app/pages/stories/stories/connexin.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            <img src='/images/stories/story-images/hotpod-yoga/benches@2x.jpg' alt='' />
+          </div>
+
+          <h2 className='u-text-heading u-color-dark-gray u-text-l u-text-light u-margin-Txl'>
+            The benefits
+          </h2>
+          <p className='para'>
+            The Connexin founders chose GoCardless due to its simplicity and ease of use.
+            “Our customers were finding the old system difficult to navigate,” says Alex. “The sign-up process and front end design of GoCardless gives them a much better experience.”
+
```
Comment (jacobpgn) [inline app/pages/stories/stories/connexin.js]: ✂️

File: app/pages/stories/stories/connexin.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+          </p>
+          <p className='story-page__quote'>
+            Switching to GoCardless cuts out process, frees up time, and avoids the need to employ a dedicated member of staff.
```
Comment (jacobpgn) [inline app/pages/stories/stories/connexin.js]: Should this be `cuts out processes`, `cuts out process inefficiencies` or something similar? `cuts out process` seems odd to me!

**Comment by Ken-Lo** [comment]
@sjnorth Watch out, I think this pull request was supposed to be made against the `add-second-batch-of-stories` branch rather than `dev`!

**Comment by jacobpgn** [comment]
@sjnorth hey - I think this one still had some errors in it!

File: app/pages/stories/stories/connexin.js [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+import StoryPageNew from '../story-page-new';
+
+export default class StoriesConnexin extends React.Component {
+  displayName = 'StoriesConnexin'
+
+  render() {
+    return (
+      <StoryPageNew
+        title='Our monthly bill runs are quick and easy since switching to GoCardless'
+        author='Alex Yueng'
```
Comment (Ken-Lo) [inline app/pages/stories/stories/connexin.js]: Should be Alex Yeung

File: app/pages/stories/stories/connexin.js [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+        position='Co-founder'
+        company='Connexin'
+        benefits='Huge reduction in payment processing admin, increased flexibility in payment timings, better signup experience'
+        bgimgUrl='url(/images/stories/story-images/nutmeg/hero-nutmeg.jpg)'
+        sidebarLogoSrc='/images/stories/story-logos/connexin.png'
+        sidebarServices='Business telephony services, VoIP,  fibre & data services'
```
Comment (Ken-Lo) [inline app/pages/stories/stories/connexin.js]: `VoIP,  fibre` --> `VoIP, fibre`

File: app/pages/stories/stories/connexin.js [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+  render() {
+    return (
+      <StoryPageNew
+        title='Our monthly bill runs are quick and easy since switching to GoCardless'
+        author='Alex Yueng'
+        position='Co-founder'
+        company='Connexin'
+        benefits='Huge reduction in payment processing admin, increased flexibility in payment timings, better signup experience'
+        bgimgUrl='url(/images/stories/story-images/nutmeg/hero-nutmeg.jpg)'
```
Comment (Ken-Lo) [inline app/pages/stories/stories/connexin.js]: Do we have a new hero image from the UX team?

File: app/pages/stories/stories/connexin.js [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+        sidebarLogoSrc='/images/stories/story-logos/connexin.png'
+        sidebarServices='Business telephony services, VoIP,  fibre & data services'
+        sidebarEmployees='23 employees'
+        sidebarPreviousProvider='CashBacs'
+        sidebarTimeSaved='12 days per year'>
+        <div className='story-body'>
+          <h2 className='u-text-heading u-color-dark-gray u-text-l u-text-light u-margin-Tn'>
+            From humble beginnings to international reach
+          </h2>
+          <p className='para u-margin-Tn'>
```
Comment (Ken-Lo) [inline app/pages/stories/stories/connexin.js]: There shouldn't be a `u-margin-Tn` class on this paragraph (it's only for the first element within the `story-body` div

**Pull request merged**